x-pack/plugins/security_solution/public/attack_discovery/pages/upgrade/index.tsx

```diff
@@ -9,9 +9,14 @@ import { AssistantAvatar, UpgradeButtons, useAssistantContext } from '@kbn/elast
 import { EuiEmptyPrompt, EuiFlexGroup, EuiFlexItem, EuiSpacer, EuiText } from '@elastic/eui';
 import React, { useMemo } from 'react';
 
+import type { ProductTier } from '../../../common/components/landing_page/onboarding/configs';
 import * as i18n from './translations';
 
-const UpgradeComponent: React.FC = () => {
+interface Props {
+  productTier: ProductTier | undefined;
+}
+
+const UpgradeComponent: React.FC<Props> = ({ productTier }) => {
```

Comment:
I don't think this component is used anymore. This file and the test can be cleaned as well.